charts/kube-prometheus-stack/templates/prometheus/clusterrole.yaml

```diff
@@ -22,7 +22,7 @@ rules:
   resources:
   - ingresses
   verbs: ["get", "list", "watch"]
-- nonResourceURLs: ["/metrics", "/metrics/cadvisor"]
+- nonResourceURLs: ["/metrics", "/metrics/slis", "/metrics/resources"]
```

Comment:
Was this a mistake, that you are removed `/metrics/cadvisor` here?

Reply:
As I mentioned above, cAdvisor permissions are controlled by the `node/metrics` (sub)`resources`, not the `nonResourceURLs` `/metrics/cadvisor`, so it can be gotten even if `/metrics/cadvisor` is removed from `ClusterRole`.